using-mobile-extensions/mobile-core/signal/README.md

```diff
@@ -0,0 +1,211 @@
+# Signal
+
+The Signal framework is bundled with [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/) and enables your app with Adobe's Experience Cloud ID service. This extension allows you to send data third-party endpoints as postback via GET and POST requests. Signals are configured by using rules in Launch. 
+
+By leveraging the same triggers and traits that you use to display an in-app message, you can configure the SDK to send customized data to a third-party destination. Also, with the appropriate permissions and configurations, you can use the `collectPii` API to send PII to a remote server.
+
+Here are the implications of the privacy status for this extension:
+
+- When the `global.privacy` configuration is set to `optedout`, the Signals extension clears all queued hits and drops future network send requests.
+- If the configuration is `optunknown`, the network sends requests that are queued but not sent out.
+- If the configuration is `optedin`, the network requests to a third-party destination are allowed, provided the trigger conditions match.
+
+To get started with Identity, complete the following steps:
+
+1. Add the **Signal** framework to your app.
+2. Define the necessary rules in Adobe Experience Platform Launch. 
+3. Implement the SDK APIs to send PII data to external destinations.
```

Comment:
Don't think you need to say "SDK APIs" --> please change to either one or the other.